packages/cli/src/databases/entities/WorkflowHistory.ts

```diff
@@ -0,0 +1,28 @@
+import { Column, Entity, ManyToOne, PrimaryColumn } from 'typeorm';
+import { jsonColumnType } from './AbstractEntity';
+import { IConnections } from 'n8n-workflow';
+import type { INode } from 'n8n-workflow';
+import { WorkflowEntity } from './WorkflowEntity';
+
+@Entity()
+export class WorkflowHistory {
+	@PrimaryColumn()
+	versionId: string;
```

Comment:
We have an older `versionId` on `WorkflowEntity` and another here. Can this cause issues or confusion? From what I remember, the older `versionId` is to prevent multiple users from overriding each others' changes. Worth considering how will they work together or whether we can unify.

Reply:
Well the `versionId` here is actually supposed to be the same string used in the workflow entity. I don't think we should unify because this one is the key to allowing users to roll back to a previous version.

This table should contain multiple entries for the same `workflowId` with different `versionId`s.